marklogic-client-api/src/main/java/com/marklogic/client/datamovement/JacksonCSVSplitter.java

```diff
@@ -39,7 +42,13 @@
     private CsvSchema csvSchema = null;
     private CsvMapper csvMapper;
     private long count = 0;
+    private ArrayNode headers = null;
+    private JsonNode csvHeaderNode;
```

Comment:
Is there a known use of csvHeaderNode?  If not, it might be best to remove the field and getter so as to maximize encapsulation.